condor-ap/condor_meter

```diff
@@ -189,8 +191,41 @@ def get_num_procs(job_ad):
 
     return procs
 
+
+def become_condor(condor_service: str = ""):
+    """condor_meter is intended to run as a SchedD cron, i.e. as the 'condor' user.
+    condor_service: if 'htcondor-ce' determine the condor user for HTCondor-CE, otherwise use the default condor config
+    """
+    try:
+        if condor_service == 'htcondor-ce':
+            os.environ.setdefault("CONDOR_CONFIG", "/etc/condor-ce/condor_config")
+            htcondor.reload_config()
+
+        condor_uid, condor_gid = htcondor.param['CONDOR_IDS'].split(".")
```

Comment:
```suggestion
        try:
            condor_uid, condor_gid = htcondor.param['CONDOR_IDS'].split(".", 1)
            condor_uid, condor_gid = int(condor_uid), int(condor_gid)
        except (IndexError, ValueError):
            raise utils.InternalError("ERROR: malformed CONDOR_IDS setting")
```
at the very least you'll need the int conversion.  You might also want to handle CONDOR_IDS being defined but empty.

Reply:
Oops, I didn't see that you made a suggestion in the email. I did the bare split on `.`:  I'd rather throw the unpack `ValueError` if there are more than one `.` separator